src/main/resources/api-specs/order-openapi.yaml

```diff
@@ -0,0 +1,214 @@
+          content:
+            application/json:
+              schema:
+                type: "array"
+                items:
+                  $ref: "#/components/schemas/AddedOrder"
+        "401":
+          description: "Unauthorized access."
+        "404":
+          description: "Orders list is empty."
+        "500":
+          description: "Internal server error."
+
+    post:
+      tags:
+        - "Orders"
+      summary: "Add order for the authorized user"
+      operationId: "addOrder"
+      requestBody:
+        description: "Payload containing a list of items to be added to the order."
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/OrderDto"
+      responses:
+        "200":
+          description: "Order placed successfully."
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/AddedOrder"
+        "400":
+          description: "Invalid input, object invalid."
+        "401":
+          description: "Unauthorized access."
+        "500":
+          description: "Internal server error."
+
+components:
+  schemas:
+    OrderStatus:
+      type: string
+      description: "Order status: 'CREATED' - new order recently created, 'DELIVERY' - order was sent, 'FINISHED' - order was delivered"
+      enum: [ CREATED, DELIVERY, FINISHED ]
+
+    OrderDto:
+      type: "object"
+      description: "An order structure in the Iced-Latte system"
+      required:
+        - items
+        - deliveryCost
+        - taxCost
+        - deliveryInfo
+        - recipientName
+        - recipientSurname
+        - email
+        - phoneNumber
+      properties:
+        items:
+          type: "array"
+          items:
+            $ref: '#/components/schemas/OrderItemDto'
+        deliveryCost:
+          type: "number"
+          format: "decimal"
+        taxCost:
+          type: "number"
+          format: "decimal"
+        deliveryInfo:
+          type: "string"
+          description: "Order details, submitted to DB on the checkout stage"
+        recipientName:
+          type: "string"
+          description: "May be different from the user's name"
+        recipientSurname:
+          type: "string"
+          description: "May be different from the user's surname"
+        email:
+          type: "string"
+          description: "May be different from the user's email"
+        phoneNumber:
+          type: "string"
+          description: "May be different from the user's phone"
+
+    OrderItemDto:
+      type: "object"
+      description: "An order item structure in the Iced-Latte system"
+      properties:
+        productId:
+          type: "string"
+          format: "uuid"
+        productQuantity:
+          type: "integer"
+          format: "int32"
+          minimum: 1
+      required:
+        - productId
+        - productQuantity
+
+    AddedOrder:
```

Comment:
name is awful and not clear

Reply:
Renamed in [46e65b4](https://github.com/Sunagatov/Iced-Latte/pull/217/commits/46e65b4c9646349a131383ace42afdeddba4b663)